src/components/Table/TableBody/TableBodyRow/TableBodyRow.story.jsx

```diff
@@ -54,11 +54,23 @@ storiesOf('Watson IoT/TableBodyRow', module)
       isExpanded={boolean('isExpanded', false)}
```

Comment:
Do we still need these stories?

Reply:
This PR was just to address RTL issues. I agree stories should be pared down but that work should happen in another PR linked to another issue.